### PR TITLE
Misc Merge [4 components+]

### PR DESCRIPTION
**Pull request by cbabnik**
I worked on a few things and didn't completely untangle the commit order, so I am putting them together as one pull request.

`utility` component and tests. Includes `-roll` `-random` `-`
`audio` component which has `-play`
`pictures` component which has `-burger`
`help` component which has help messages for most commands

Eslint is also set up in the PR

Some infrastructure is also modified, Scan now returns 1 command only, and actor accepts image URLS.

### CODE REVIEW

File: components/example.js [hunk 1]
```diff
@@ -1,75 +1,77 @@
...
-        this.addCommand('\\+play$', () => this.playAudio("sample.mp3"));
-        this.addCommand('\\+coinflip$', () => this.coinflip("", ""));
-        this.addCommand('\\+coinflip (\\S+) (\\S+)$', this.coinflip);
-        this.addCommand('\\+coinflip "(.*)" "(.*)"$', this.coinflip);
+        this.addCommand(/\+win$/, this.win);
+        this.addCommand(/\+countdown (\d+)$/, this.countdown);
+        this.addCommand(/\+countdown$/, () => this.countdown(5));
+        this.addCommand(/\+delay (\d+) (.*)$/, this.delay);
+        this.addCommand(/\+spam (\d+) (.*)$/, this.spam);
+        this.addCommand(/\+alarm \[(.*)] (.*)$/, this.alarm);
```
Comment (KHartwig) [inline components/example.js]: Is this missing a backslash before the ']' ?

File: components/utility.js [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+        if (metaInfo.commandMatchesCount === 1) {
+            this.setAction('message', 'Try `+random [a] [b] [c]`');
+        }
+    }
+    randomInfoB() {
+        this.setAction('message', 'You need more than one option to random between');
+    }
+
+    random(options) {
+        if(options.includes(' ')) {
```
Comment (KHartwig) [inline components/utility.js]: space after if

File: components/utility.js [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+    }
+
+    roll(min, max) {
+        const result = bigInt.randBetween(min,max);
+        this.setAction('message', result.toString());
+    }
+
+    randomInfo(metaInfo) {
+        if (metaInfo.commandMatchesCount === 1) {
+            this.setAction('message', 'Try `+random [a] [b] [c]`');
```
Comment (KHartwig) [inline components/utility.js]: Should be 'Try -random ...' seeing this elsewhere in the file too

Comment (KHartwig) [reply components/utility.js]: like '+roll' in the rollInfo, should be -roll

File: components/example.js (hunk 1 above)
Comment (cbabnik) [reply components/example.js]: Surprisingly `]` only needs to be escaped if you already used `[`

File: components/utility.js (hunk 2 above)
Comment (cbabnik) [reply components/utility.js]: I'll add eslint rule "keyword-spacing".

Whenever we see something with inconsistent or ugly style, lets try to find an eslint rule for it.

Comment (cbabnik) [reply components/utility.js]: Correction: the eslint is `space-after-keywords`

**Pull request merged**